dataland-frontend/src/components/pages/CompanyDataRequestsOverview.vue

```diff
@@ -0,0 +1,477 @@
+                        </PrimeButton>
+                      </div>
+                    </div>
+                  </template>
+                </Column>
+              </DataTable>
+            </div>
+          </div>
+        </div>
+        <div v-if="!waitingForData && storedDataRequests.length == 0">
+          <div class="d-center-div text-center px-7 py-4">
+            <p class="font-medium text-xl">You have no data requests yet.</p>
+          </div>
+        </div>
+      </TheContent>
+    </DatasetsTabMenu>
+    <TheFooter :is-light-version="true" :sections="footerContent" />
+  </AuthenticationWrapper>
+</template>
+
+<script lang="ts">
+import TheFooter from '@/components/generics/TheNewFooter.vue';
+import contentData from '@/assets/content.json';
+import type { Content, Page } from '@/types/ContentTypes';
+import TheContent from '@/components/generics/TheContent.vue';
+import TheHeader from '@/components/generics/TheHeader.vue';
+import { defineComponent, inject, ref } from 'vue';
+import type Keycloak from 'keycloak-js';
+import { ApiClientProvider } from '@/services/ApiClients';
+import DataTable from 'primevue/datatable';
+import Column from 'primevue/column';
+import {
+  frameworkHasSubTitle,
+  getFrameworkSubtitle,
+  getFrameworkTitle,
+  humanizeStringOrNumber,
+} from '@/utils/StringFormatter';
+import DatasetsTabMenu from '@/components/general/DatasetsTabMenu.vue';
+import { convertUnixTimeInMsToDateString } from '@/utils/DataFormatUtils';
+import {
+  type CompanyRoleAssignment,
+  RequestStatus,
+  AccessStatus,
+  type StoredDataRequest,
+} from '@clients/communitymanager';
+import InputText from 'primevue/inputtext';
+import FrameworkDataSearchDropdownFilter from '@/components/resources/frameworkDataSearch/FrameworkDataSearchDropdownFilter.vue';
+import { type FrameworkSelectableItem, type SelectableItem } from '@/utils/FrameworkDataSearchDropDownFilterTypes';
+import { FRAMEWORKS_WITH_VIEW_PAGE } from '@/utils/Constants';
+import { getFrontendFrameworkDefinition } from '@/frameworks/FrontendFrameworkRegistry';
+import AuthenticationWrapper from '@/components/wrapper/AuthenticationWrapper.vue';
+import { accessStatusBadgeClass, badgeClass } from '@/utils/RequestUtils';
+import PrimeButton from 'primevue/button';
+
+export default defineComponent({
+  name: 'MyDataRequestsOverview',
+  computed: {
+    AccessStatus() {
+      return AccessStatus;
+    },
+  },
+  components: {
+    PrimeButton,
+    AuthenticationWrapper,
+    FrameworkDataSearchDropdownFilter,
+    DatasetsTabMenu,
+    TheFooter,
+    TheContent,
+    TheHeader,
+    DataTable,
+    Column,
+    InputText,
+  },
+
+  setup() {
+    return {
+      frameworkFilter: ref(),
+      datasetsPerPage: 100,
+      getKeycloakPromise: inject<() => Promise<Keycloak>>('getKeycloakPromise'),
+      companyRoleAssignments: inject<Array<CompanyRoleAssignment>>('companyRoleAssignments'),
+    };
+  },
+
+  data() {
```

Comment:
Das klicken einer Zeile in der Tabelle führt aktuell zu nichts. Sollte da nicht die Detailübersicht des requests kommen?